layouts/index.redirects

```diff
@@ -10,6 +10,7 @@ docs/v3.4.0/reporting-bugs    /docs/v3.4.0/reporting_bugs/
 
 /docs/next    /docs/v3.5/
 /docs/next/*    /docs/v3.5/:splat
+/docs/v3.5/platforms/aws/	/blog/2021/etcd-deployments-on-aws/
```

Comment:
We need to change this redirect now that the blog post has moved.

Otherwise, looks good to me!